blockchain/fundamentals/ethereum-on-the-web/metamask.md

```diff
@@ -0,0 +1,57 @@
+---
+author: mihaiberq
+
+levels:
+
+  - beginner
+
+  - basic
+
+type: normal
+
+category: must-know
+
+standards:
+  blockchain.identify-blockchain-applications.4: 10
+
+aspects:
+
+  - introduction
+
+  - workout
+
+---
+# MetaMask
+
+---
+## Content
+
+Like for bitcoins, you need a wallet to be able to hold any ether. The Ethereum client works well if you are looking to also run a node. In this workout we'll discuss some alternatives.
+
+**MetaMask** is a browser extension that allows you to interact with the Ethereum network without having to download the entire chain. Moreover, it supports any ERC20 token (standards for the win!) alongside ETH.
+
+It is a simple interface that stores any generated keys offline, on your computer. With MetaMask, you can use Ethereum dapps in any browser that supports the extension.
+
+---
+## Practice
+
+What currencies can Metamask keep track of?
+
+???
+
+* Ethereum and ERC20
+* Ethereum and Bitcoin
+* Ethereum, ERC20 and Bitcoin
+* Ethereum
+
+---
+## Revision
+
+Where does Metamask store your wallet?
```

Comment:
so the generated keys are your wallet?

Reply:
yep, discussed everything wallet-related in a dedicated workout